index.js

```diff
@@ -31,6 +32,22 @@ const argv = require('yargs')
         desc: 'Print an HTML file or URL to PDF',
         builder: {
             ...commonOptions,
+            'emulatemedia': {
+                string: true,
+                default: ''
+            },
+            'injectjs': {
```

Comment:
rename to `inject-js` to be consistent with other multi-term options (will materialize as `argv.injectJs` then)

Reply:
done